src/NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestIsUpgradableCapableLayoutRenderer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Http.Features;
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// Indicates if the server can upgrade this request to an opaque, bidirectional stream.
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-request-is-upgradable}
+    /// </remarks>
+    [LayoutRenderer("aspnet-request-is-upgradable")]
```

Comment:
Still not happy about this name. Maybe include include the word "stream" and "bidirectional"

Reply:
Complete

Reply:
Completed